code/__DEFINES/admin.dm

```diff
@@ -102,7 +102,7 @@
 #define STICKYBAN_ROGUE_CHECK_TIME 5
 
 
-#define POLICY_POLYMORPH "polymorph" //Shown to vicitm of staff of change and related effects.
+#define POLICY_POLYMORPH "policy_polymorph" //Shown to vicitm of staff of change and related effects.
```

Comment:
Undocumented/unrelated change?

Reply:
Oh that was from me testing stuff, yeah unrelated.

Reply:
There, removed.